trade-finance-manager-api/api-tests/v1/deal-cancellation/deal-cancellation.get.api-test.ts

```diff
@@ -68,13 +67,6 @@ describe('/v1/deals/:id/cancellation', () => {
         jest.resetAllMocks();
       });
 
-      withTeamAuthorisationTests({
```

Comment:
Why has this been removed please?

Reply:
The GET endpoints don't need to be behind RBAC, since all users need to know if a deal has been scheduled